updatem/orchestration/update_orchestrator_feedback.go

```diff
@@ -116,32 +131,28 @@ func handleDomainIdentificationFailed(orchestrator *updateOrchestrator, domain,
 	orchestrator.updateStatusIdentificationFailed(domain, message)
 }
 
-func handleDomainCompletedEvent(orchestrator *updateOrchestrator, domain, message string) {
-	if orchestrator.operation.status != types.StatusRunning {
-		return
-	}
+func handleDomainCompletedEvent(orchestrator *updateOrchestrator, domain, message string, actions []*types.Action) {
 	domainStatus := orchestrator.operation.domains[domain]
-	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete {
+	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete ||
+		domainStatus == types.BaselineStatusCleanupSuccess || domainStatus == types.BaselineStatusCleanupFailure {
 		return
 	}
-	orchestrator.operation.domains[domain] = types.StatusCompleted
-	orchestrator.command(context.Background(), orchestrator.operation.activityID, domain, types.CommandCleanup)
+	orchestrator.operation.domains[domain] = types.BaselineStatusCleanupSuccess
+	orchestrator.domainUpdateCompleted()
 }
 
-func handleDomainIncomplete(orchestrator *updateOrchestrator, domain, message string) {
-	if orchestrator.operation.status != types.StatusRunning {
-		return
-	}
+func handleDomainIncomplete(orchestrator *updateOrchestrator, domain, message string, actions []*types.Action) {
 	domainStatus := orchestrator.operation.domains[domain]
-	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete {
+	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete ||
+		domainStatus == types.BaselineStatusCleanupSuccess || domainStatus == types.BaselineStatusCleanupFailure {
 		return
 	}
 	orchestrator.operation.delayedStatus = types.StatusIncomplete
-	orchestrator.operation.domains[domain] = types.StatusIncomplete
-	orchestrator.command(context.Background(), orchestrator.operation.activityID, domain, types.CommandCleanup)
+	orchestrator.operation.domains[domain] = types.BaselineStatusCleanupFailure
```

Comment:
The function is called _handleDomainIncomplete_. Is _types.BaselineStatusCleanupFailure_ the correct domain status?